src/nvim/terminal.c

```diff
@@ -1155,7 +1155,6 @@ static void redraw(bool restore_cursor)
     save_col = ui_current_col();
   }
   block_autocmds();
-  validate_cursor();
```

Comment:
Shouldn't be needed since we call `mb_check_adjust_col()` in `adjust_topline()`.